build.gradle

```diff
@@ -53,6 +53,8 @@ task functional(type: Test) {
   description = "Runs Functional Tests"
   testClassesDirs = sourceSets.functionalTest.output.classesDirs
   classpath = sourceSets.functionalTest.runtimeClasspath
+
+  environment("APPINSIGHTS_INSTRUMENTATIONKEY", "some-key")
```

Comment:
It should be the other key for Java apps - `APPLICATION_INSIGHTS_IKEY`

Reply:
initially it used to be. please read the description - we are aligning nodejs and java apps regarding app insights usage